### PR TITLE
ci(precommit): update

### PR DESCRIPTION
**Pull request by xsc27**

### CODE REVIEW

**Comment by xsc27** [comment]
🤔 but i'm not a first-time contributor. hmm, i guess my commits have been squashed out of existence.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,6 +23,29 @@ env:
   PYTHONIOENCODING: "UTF-8"
 
 jobs:
+  preflight:
```
Comment (Yajo) [inline .github/workflows/ci.yml:26]: https://github.com/copier-org/copier/blob/600ebdb4920c6c535c6e856d9b3dc504160949e4/tests/test_tools.py#L10

This is already being tested. This part is not needed.

File: copier/errors.py [hunk 2]
```diff
@@ -90,7 +89,9 @@ class CopierAnswersInterrupt(CopierError, KeyboardInterrupt):
 
     """
 
-    def __init__(self, answers: 'AnswersMap', last_question: 'Question', template: 'Template') -> None:
+    def __init__(
+        self, answers: "AnswersMap", last_question: "Question", template: "Template"
```
Comment (Yajo) [inline copier/errors.py:93]: Why this change if you're not updating black?

**Review by Yajo — CHANGES_REQUESTED** [review]

File: copier/errors.py (hunk 2 above)
Comment (xsc27) [reply copier/errors.py:93]: 🤔 this was a result of me running `pre-commit run -a`

Comment (xsc27) [reply copier/errors.py:93]: well i can drop all the commits except bumping the pre-commit hooks w/ `pre-commit autoupdate`

Comment (xsc27) [reply copier/errors.py:93]: ☝🏽 🧐  How is pre-commit/black passing since those single quotes?

**Comment by xsc27** [comment]
`pre-commit run -a`:

```
black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted copier/errors.py
All done! ✨ 🍰 ✨
1 file reformatted, 38 files left unchanged.

isort....................................................................Failed
- hook id: isort
- files were modified by this hook

Fixing /Users/carlos.meza/git/GitHub/copier/copier/errors.py

prettier.................................................................Failed
- hook id: prettier
- files were modified by this hook

CHANGELOG.md
.github/workflows/ci.yml
```

File: copier/errors.py (hunk 2 above)
Comment (Yajo) [reply copier/errors.py:93]: I think this is an important point. Maybe the test is buggy?

Comment (Yajo) [reply copier/errors.py:93]: Indeed, fixed in https://github.com/copier-org/copier/pull/511

**Comment by Yajo** [comment]
This has conflicts. Do you have plans on continuing with this PR?

Closing just to keep the list clean, but feel free to reopen if so. Thanks!
